jest/globals.js

```diff
@@ -6,6 +6,6 @@ global.Dispatcher = () => {
     return {}
 };
 
-global.$ = require("jquery")
-global.Vue = require("vue")
```

Comment:
Require are no longer a thing in Vite but. I do wonder if we need the global Vue object instance to be available since some packages , or repos use it? Maybe? Worth a research